src/main/java/com/commercetools/sync/services/impl/BaseService.java

```diff
@@ -116,7 +117,9 @@
         final String draftKey = keyMapper.apply(draft);
 
         if (isBlank(draftKey)) {
-            syncOptions.applyErrorCallback(format(CREATE_FAILED, draftKey, "Draft key is blank!"));
+            syncOptions.applyErrorCallback(
```

Comment:
@heshamMassoud Does it make sense to call applyErrorCallback on sync validations? Especially when for similar issue we call warningCallback: https://github.com/commercetools/commercetools-sync-java/pull/367/files#diff-38b71d2c035833260351074420bf8067R60

Reply:
Hi @butenkor is the reply from @heshamMassoud clear or are there still points that need to be checked?